tests/MessageTest.php

```diff
@@ -1057,4 +1057,15 @@ public function testGbkCharsetDecoding(): void
 
         static::assertSame('Hi', \trim($message->getDecodedContent()));
     }
+    
+    public function testHtmlMultipartMessage(): void
+    {
+        $this->mailbox->addMessage($this->getFixture('HTMLmultipartMessage'));
+
+        $message = $this->mailbox->getMessage(1);
+        
+        static::assertSame('<html>', $message->getBodyHtml());
```

Comment:
I'm sorry, but this is not a game of copy-paste, I don't know your expectations, it is up to you to write them in a way the bug you reported could emerge.

Reply:
Hi,

first of all thanks for your patience. As I pointed out I'm new to this, so I simply didn't know that saving the file updates the pull request. Sorry for that.

I hope the asserts are correct now.

Kind regards
Kevin

Reply:
Your test is now clear and self-contained.

Finding the solution is another story though...

Reply:
Just out of curiosity, can you explain why this is happening?